Points-Thing App

selvarun-umass opened the pull request:
The commit changes include configuration of the API Points table form and the setup of a new UI Policy to make certain fields mandatory.

SapphicFire commented on 9c7718cddb229150791d8f8d13961979/update/sys_db_object_d47a3b43473d35104fb86285d36d4380.xml:
```diff
@@ -9,30 +9,27 @@
         <sys_created_on>2022-10-13 03:05:34</sys_created_on>
-        <sys_id>d4ba90c1db629150791d8f8d139619cb</sys_id>
-        <sys_mod_count>0</sys_mod_count>
+        <sys_id>d47a3b43473d35104fb86285d36d4380</sys_id>
```
Please clarify why the sys_id is changing. This will be a fundamentally different record which will drop the records from this table as the table itself drops on an install. Additionally, any references will break

SapphicFire reviewed (changes requested):
Thank you for your contribution. This Pull Request contains changes that seem to be breaking or altering the existing functionality by changing the sys_id of the API Points table. Please provide a thorough justification for these changes, including the reasons and the impact. Also, consider discussing breaking changes in an issue before submitting a pull request to ensure alignment with the project’s goals and avoid potential conflicts. Closing this for now. Once you make additional changes, feel free to re-open this Pull Request or create a new one.

selvarun-umass commented:
> Thank you for your contribution. This Pull Request contains changes that seem to be breaking or altering the existing functionality by changing the sys_id of the API Points table. Please provide a thorough justification for these changes, including the reasons and the impact. Also, consider discussing breaking changes in an issue before submitting a pull request to ensure alignment with the project’s goals and avoid potential conflicts. Closing this for now. Once you make additional changes, feel free to re-open this Pull Request or create a new one.

Oh :( I will take a look at it. Thank you! There is nothing written in the contribution section like the one we have for the Plants folder. Hence, I checked with my colleague who previously participated and suggested modifying the app. I modified the API table form configuration and added a UI policy to mandate a few fields. Could you please suggest what kinds of contributions are eligible for a pull request? I have created the app on Slack, and the points system is working as expected. Is this what you are looking for? Can I commit those API Points Table records and submit them? This is my first time participating in the event, and I want to contribute sincerely. Could you please let me know how I can reopen this pull request? Thank you so much for understanding and helping me out.

selvarun-umass commented on 9c7718cddb229150791d8f8d13961979/README.md:
Oh :( I will take a look at it. Thank you! There is nothing written in the contribution section like the one we have for the Plants folder. Hence, I checked with my colleague who previously participated and suggested modifying the app. I modified the API table form configuration and added a UI policy to mandate a few fields. Could you please suggest what kinds of contributions are eligible for a pull request? I have created the app on Slack, and the points system is working as expected. Is this what you are looking for? Can I commit those API Points Table records and submit them? This is my first time participating in the event, and I want to contribute sincerely. Could you please let me know how I can reopen this pull request? Thank you so much for understanding and helping me out.

selvarun-umass replied:
@earlduque, could you please help me with the above information? Thank you so much in advance!
